### DISCUSSION TITLE
Usage Analytics

### DISCUSSION THREAD

**Discussion by rmccreath** [rstudio/learnr#687] [Q&A]
Hey, I've developed a number of `learnr` apps to start a suite of Data Science training for Public Health Scotland. Happy to share a repo of some intro courses, and welcome feedback (https://github.com/Public-Health-Scotland/learnr-online). These have been so well received but this reception has only been captured through optional feedback or ad-hoc conversations. In order to include in performance reports and make a case in business planning, more detailed usage analytics are moving from a "nice to have" to a "need to have". I'd like to start with some basic Google Analytics and perhaps build a Shiny dashboard on top of that data but I'm encountering issues getting any data from these apps into Google, I believe this to be around the prerendered nature of the `learnr` apps. Does anyone have any experience they could share to help me get going? Thanks in advance 😄 

**TLDR: add Google Analytics to `learnr` apps.**

**Comment by gadenbuie** [comment]
This recent RStudio Enterprise Meetup might be relevant and helpful, since learnr tutorials are Shiny apps that can be deployed to Connect: [Shiny Usage Tracking in RStudio Connect - YouTube](https://www.youtube.com/watch?v=0iljqY9j64U)

**Reply by rmccreath** [reply]
Thanks, @gadenbuie - this is great. We're currently building our infrastructure to include RStudio Connect but it won't be live until the end of this year. I was looking to do something in the meantime to get traffic/engagement insights. I imagine it's me that's doing something wrong 😄
